shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -500,6 +518,10 @@
         // Get the intersection rect between the current rect
         // and the platform view rect.
         joined_rect.intersect(platform_view_rect);
+        // Subpixels in the platform may not align with the canvas subpixels.
+        // To workaround it, round the rect values.
+        joined_rect.setLTRB(std::round(joined_rect.left()), std::round(joined_rect.top()),
```

Comment:
Are you sure rounding is what we want, instead of cutting off the subpixels?  Could we end up with a row or column of pixels whose content is empty?

Reply:
It nows rounds the `left` and `top` bounds to the nearest `int`, whereas the `right` and bottom bounds are ceiled. This ensures that the rect is slightly larger than the original one.